openhgnn/dataset/NodeClassificationDataset.py

```diff
@@ -505,7 +505,7 @@ class OGB_NodeClassification(NodeClassificationDataset):
     def __init__(self, dataset_name, *args, **kwargs):
         super(OGB_NodeClassification, self).__init__(*args, **kwargs)
         if dataset_name == 'ogbn-mag':
-            dataset = DglNodePropPredDataset(name='ogbn-mag')
+            dataset = DglNodePropPredDataset(name='ogbn-mag', root='openhgnn/dataset/data/')
```

Comment:
Do not specify a fixed path. You will block the other users who want to modify the download path!